### DISCUSSION TITLE
I fail to see the point

### DISCUSSION THREAD

**Discussion by WereCatf** [community/community#13548] [Other Feature Feedback, Questions, & Ideas]
I don't use Github for socialization or "community", I use it as a code-repository and to access/contribute to projects that I use. I don't see why I'd care about random people or their doings, or random projects for that matter, if I'm not using them myself.

"But but but DISCOVERABILITY!!" -- Yeah, no. We're not talking about new movies or TV-shows or such, we're talking about code; if you have an active need for something, you're likely to use the search and once you have found something that works for your needs, why would you want recommendations for more of the same? I mean, your need has already been fulfilled at that point, so recommendations for more of the same is now clutter, not useful information.

Sure, there will be some people who like this feature, but I would hazard a guess that most people won't care.

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
